.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "hoomd/extern/libgetar"]
 	path = hoomd/extern/libgetar
-	url = https://bitbucket.org/glotzer/libgetar
+	url = https://github.com/glotzerlab/libgetar
```

Comment:
I'm not sure how submodules work - I tried `git submodule update --init` after this change, but running `git remote -v` in the `extern/libgetar` directory still gives Bitbucket links.

Also it might be helpful to update this submodule to the latest commit on GitHub, so that its own internal docs and links are to GitHub as well.